### ISSUE TITLE
Support for custom csslint rules

### ISSUE THREAD

**Issue by cvn** [jgable/grunt-lesslint#16]
It is possible to add custom rules to csslint. Is it possible to do this with grunt-lesslint? 

Here are some examples of rules that I would like to add:
https://github.com/drBenway/CSSLint-custom-rules

The author suggests adding these manually to the csslint-node.js file, but I'd prefer to have the rules be pluggable. Otherwise I'd have to fork csslint and grunt-lesslint to do it.

**Comment by davidparsson** [comment]
Any news on this? If I were to submit a PR that implements this, would that be included and released any time soon?

**Comment by jgable** [comment]
Sure, give it a shot and I'll make you an admin on the project if you want.

**Comment by davidparsson** [comment]
Thanks! I'll see what I can do.

**Comment by davidparsson** [comment]
I've managed to load rule files with Grunt style globbing patterns, like this:
```coffeescript
lesslint:
  options:
    customRules: ['rules/**/*.coffee']
  src: ['less/**/*.less']
```

I'm however facing trouble when it comes to allowing different `customRules` configurations for different lesslint targets. This is because CSSLint is a _singleton_ and as far as I can tell it provides no simple way to reset its state.

It would be possible to resolve this using a rather complex solution, but the easiest and least error-prone implementation would be to only allow users to configure the same set of custom rules for _all_ targets. In other words, only allowing `customRules` under the task's global `option` block, not under targets' `option` blocks.

@jgable, what are your thoughts on this? If you have the time, please [review my changes](https://github.com/davidparsson/grunt-lesslint/commit/8b1fd57bc0a8bd2c718c262ecb16f87f409ff8af) so far.

**Comment by jgable** [comment]
I took a look today and the changes look good.  It's unfortunate we can't remove the specific rules we add in for each configuration section.  

I found [this issue related to removeRule](https://github.com/CSSLint/csslint/pull/215) that points to us being able to pass the explicit rules to the `.verify` call.  It looks like we'd have to refactor our call to verify to always pass the default rules and then custom rule ids of the files matched in the `customRules` option.
